backend/Controllers/Activity.go

```diff
@@ -0,0 +1,122 @@
+package routeControllers
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/benzend/goalboard/database"
+	"github.com/benzend/goalboard/models"
+	"github.com/benzend/goalboard/utils"
+	_ "github.com/lib/pq"
+)
+
+var body ActivityProgress
+
+var timestamp = time.Now().Format(time.RFC3339)
+
+// Connect to the database
+var db, err = database.Connect()
```

Comment:
i wonder if we should just use db within the context? should be an already existing db reference.

Reply:
@benzend Correct I do need to go through and still do this, as I want to add time out, I just have to finish the other half the refactor first. Just didn't quite get this far however will make sure this gets put in.